include/aws/common/byte_buf.h

```diff
@@ -61,7 +65,10 @@ int aws_byte_buf_init(struct aws_allocator *allocator, struct aws_byte_buf *buf,
  * Returns AWS_OP_SUCCESS in case of success or AWS_OP_ERR when memory can't be allocated.
  */
 AWS_COMMON_API
-int aws_byte_buf_init_copy(struct aws_allocator *allocator, struct aws_byte_buf *dest, const struct aws_byte_buf *src);
```

Comment:
If we're writing methods which consume from a cursor but _don't_ advance the cursor, we should document this explicitly.

Reply:
Like you said, I think accepting it by value gets the point across. I'll do that.